src/views/examples/all-components/index.njk

```diff
@@ -2,69 +2,37 @@
 
 {% block content %}
 
-  {% include "../components/breadcrumb/breadcrumb.njk" %}
+  {% include "../../../components/breadcrumbs/breadcrumbs.njk" %}
```

Comment:
I think there's a few components missing from this…

- Back link
- Date input
- Error summary
- File upload
- Previous next
- Skip link
- Table
- Tag

Reply:
I was just trying to get this page rendering as it was before, should we remove it if no one was using it?

Reply:
Fair.